components/popup/RateDish.vue

```diff
@@ -84,14 +86,20 @@ const inputNickname = useLocalStorage('rate-dish--nickname', () => '')
 
 const errorRequireStars = useState(`rate-dish-${props.occurrence.id}--error-stars`, () => false)
 const submittedLoading = useState(`rate-dish-${props.occurrence.id}--submitted-loading`, () => false)
-const readyToSubmit = computed(() => !!inputStars.value)
+const readyToSubmit = computed(() => !!inputStars.value && currentImageId.value !== 'uploading')
 
 watch(inputStars, () => errorRequireStars.value = false)
 
+const currentImageId = useState<string | null>(`rate-dish-${props.occurrence.id}--current-image-id`, () => null)
+
 const { open, onChange } = useFileDialog()
-onChange((files) => {
-  if (files?.length)
+onChange(async (files) => {
+  if (files?.length) {
     inputImage.value = files[0]
+
+    currentImageId.value = 'uploading'
```

Comment:
I don't really like having this special string value here, a separate boolean would be cleaner but it's fine this way. Maybe extract it as a constant though, maybe not idk

Reply:
done